bedrock/security/templates/security/client-bug-bounty.html

```diff
@@ -191,7 +191,7 @@ <h2 id="static-analysis-bounty">Static Analysis Bounty</h2>
 
   <p><b>Examples of Quality of Submission:</b> As mentioned, the bounty amount we grant for the query will be determined based on the quality of the submission, and an estimation of the number of issues we think it may identify in a one to three-year timespan. On the low end, if you submit a query that identifies a single historical issue of a syntactical misuse of an API we are unlikely to use in future code, we may not issue a bounty, and if we did it would be below the $2500 range. And on the high end, if you submit a query that matches 3 unknown issues today, in code written in the last year - we can expect it will identify a significant number of issues in the future and would be looking above the $7500 range. (Plus you’d be eligible for separate bounties on those 3 issues.)</p>
 
-  <p><em>Note</em>: While we previously accepted submissions of CodeQL queries, we no longer do. We still provide <a href="https://bug-bounty-codeql-databases.s3.us-east-2.amazonaws.com/index.html">CodeQL databases available for download</a> and you are of course encouraged to develop CodeQL queries if you think they will be valuable, and submit any findings you glean from them.</p>
+  <p><em>Note</em>: While we previously accepted submissions of CodeQL queries, we no longer do.You are of course encouraged to develop CodeQL queries if you think they will be valuable, and submit any findings you glean from them.</p>
```

Comment:
Reinstating missing space:

```suggestion
  <p><em>Note</em>: While we previously accepted submissions of CodeQL queries, we no longer do. You are of course encouraged to develop CodeQL queries if you think they will be valuable, and submit any findings you glean from them.</p>
```